src/scheduler/resource_resv.c

```diff
@@ -150,6 +150,7 @@ new_resource_resv()
 	resresv->can_never_run = 0;
 	resresv->is_peer_ob = 0;
 
+	resresv->is_prov_needed = 0;
```

Comment:
You need to copy the value in dup_resource_resv().

Reply:
Done.